modules/serverless-nodejs-function-return-values.adoc

```diff
@@ -22,6 +22,98 @@ function handle(context, customer) {
 }
 ----
 
+[id="serverless-nodejs-function-return-values-primitive-types_{context}"]
+== Returning primitive types
+
+Functions can return any valid JavaScript type, including primitives such as strings, numbers, and booleans:
+
+.Example function returning a string
+[source,javascript]
+----
+function handle(context) {
+  return "This function Works!"
+}
+----
+
+Calling this function returns the string:
+
+[source,terminal]
+----
+$ curl https://myfunction.example.com
+----
+
+[source,terminal]
+----
+This function Works!
+----
+
+. Example function returning a number
+[source,javascript]
+----
+function handle(context) {
+  let somenumber = 100
+  return { body: somenumber }
+}
+----
+
+Calling this function returns the number:
+
+[source,terminal]
+----
+$ curl localhost:8080
+----
+
+[source,terminal]
+----
+100
+----
+
+. Example function returning a boolean
```

Comment:
Here it seems the numbering is not working on the rendered preview. It appears as  "1." (just like the previous one) but I believe it should be rendered as "2."

Reply:
Thanks, fixed!